src/Microsoft.AspNet.Mvc.Core/Formatters/JsonOutputFormatter.cs

```diff
@@ -16,13 +17,23 @@ public class JsonOutputFormatter : OutputFormatter, IJsonOutputFormatter
         private JsonSerializerSettings _serializerSettings;
 
         public JsonOutputFormatter()
```

Comment:
> not sure how to access the MvcOptions

Model the approach after the block a couple of lines up from that link.  `Instance` and `Singleton` have the same lifetime.  So:
``` c#
services.TryAdd(ServiceDescriptor.Singleton<JsonOutputFormatter>(serviceProvider =>
{
    var options = serviceProvider.GetRequiredService<IOptions<MvcOptions>>().Options;
    return new JsonOutputFormatter(options.SerializerSettings);
}));
```